app/Testing/MocksGuzzleInstance.php

```diff
@@ -144,18 +140,4 @@ protected function assertAllTapperRequestsLike(array $expectedRequests): void
         }
     }
 
-    /**
-     * Create a real guzzle client that can use a MockHandler class to mock all
-     * requests.
-     * @see http://docs.guzzlephp.org/en/stable/testing.html
-     * @param MockHandler $mock
-     * @return Client
-     */
-    private function mockGuzzleAppInstanceWithHandler(MockHandler $mock): Client
```

Comment:
Now that I want to do this more than once, it felt reasonable to make instantiating the mocked Client Tapper's responsibility. (We didn't do this originally because we wrote Tapper **after** writing a bunch of tests with raw MockHandler)